src/__experimental__/components/radio-button/radio-button-classic.style.js

```diff
@@ -0,0 +1,57 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../style/themes';
+import { StyledCheckableInput } from '../checkable-input/checkable-input.style';
+import FieldHelpStyle from '../field-help/field-help.style';
+import HiddenCheckableInputStyle from '../checkable-input/hidden-checkable-input.style';
+import StyledCheckableInputSvgWrapper from '../checkable-input/checkable-input-svg-wrapper.style';
+import LabelStyle from '../label/label.style';
+
+export default ({
+  disabled, checked, fieldHelpInline, reverse, theme
+}) => theme.name === THEMES.classic && css`
```

Comment:
Maybe this could be refactored to use `isClassic(theme)` from `src/utils/helpers/style-helper/style-helper.js` ?

Reply:
Updated 👍